docs/netlify/utils/convert.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
```

Comment:
This file follows the convention of the 4 spaces for indentation, where the project uses 2 spaces. Can you fix that?

BTW: What is that file? Is it used anywhere? I didn't find any mention of this file in the entire project.

Reply:
This is simple script to generate .json file from the nginx .conf. Ok, I am going to document how to use it.